engine/src/main/java/org/terasology/rendering/dag/nodes/AlphaRejectBlocksNode.java

```diff
@@ -45,34 +50,57 @@
  * In alpha-blending the color of a semi-transparent fragment is combined with
  * the color stored in the frame buffer and the resulting color overwrites the previously stored one.
  */
-public class AlphaRejectBlocksNode extends WireframeCapableNode {
+public class AlphaRejectBlocksNode extends AbstractNode implements WireframeCapable {
 
     private static final ResourceUrn CHUNK_SHADER = new ResourceUrn("engine:prog.chunk");
 
     @In
     private ComponentSystemManager componentSystemManager;
 
     @In
+    private Config config;
+
+    @In
     private WorldRenderer worldRenderer;
 
     @In
     private RenderQueuesHelper renderQueues;
 
     private Camera playerCamera;
     private Material chunkShader;
+    private SetWireframe wireframeStateChange;
+    private RenderingDebugConfig renderingDebugConfig;
 
     /**
      * Initialises the node. -Must- be called once after instantiation.
      */
     @Override
     public void initialise() {
-        super.initialise();
         playerCamera = worldRenderer.getActiveCamera();
+
+        wireframeStateChange = new SetWireframe(true);
+        renderingDebugConfig = config.getRendering().getDebug();
+        new WireframeTrigger(renderingDebugConfig, this);
```

Comment:
Ok guys, I'll keep all this in mind for the next PRs.

And thank you @indianajohn for the research on the GC and the one-liner suggestion.